### DISCUSSION TITLE
Community.Toolkit.Maui Support for Microsoft.Maui.Controls 8.0.7

### DISCUSSION THREAD

**Discussion by BlueFire-LLC** [CommunityToolkit/Maui#1727] [Q&A]
Currently Community.Toolkit.Maui version 7.0.1 only supports Microsoft.Maui.Controls 8.0.6.

Is there a timeline for supporting version 8.0.7?

**Comment by BlueFire-LLC** [comment]
Now, Microsoft is up to version 8.0.10 and Community Toolkit is still dependent on 8.0.6. Is Community Toolkit no longer being updated?

**Reply by bijington** [reply]
Microsoft.Maui v8.0.10 was literally released today 😆.

We have a minimum dependency on Microsoft.Maui so you should still be able to use 8.0.10 in your app and use the toolkit. As Pedro mentioned, are you seeing any specific issues?

**Comment by pictos** [comment]
Are you facing any issues with the old version?

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
